source/whats-new.txt

```diff
@@ -0,0 +1,34 @@
+==========
+What's New
+==========
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Learn what's new in:
+
+
+* :ref:`Version 2.19 <version-2.19>`
+
+.. _upcoming-breaking-changes:
+
+.. _version-2.19:
+
+What's New in 2.19 
+-------------------
+
+The 2.19 driver release includes the following new features:
+
+- The ``ObjectSerializer`` allows deserialization of only safe types, as determined by 
+  the configurable ``AllowedTypes`` function. 
+- Changed the default ``LinqProvider`` to LINQ3.
+- Added builders for creating Atlas searches.
```

Comment:
- Added Atlas Search (``$search``) support to builders, aggregation pipelines, and LINQ3